internal/security/grantvmgroupaccess.go

```diff
@@ -68,11 +73,11 @@ const (
 // include Grant ACE entries for the VM Group SID. This is a golang re-
 // implementation of the same function in vmcompute, just not exported in
 // RS5. Which kind of sucks. Sucks a lot :/
-func GrantVmGroupAccess(name string) error {
+func GrantVmGroupAccess(name string, desiredAccess AccessMask) error {
```

Comment:
Not a huge deal but why are we not consuming this from go-winio and enhancing there? Seems odd to move to an internal model

Reply:
Also to avoid the other breaks you may want to consider an overload `GrantVmGroupAccessWithMask` or something

Reply:
@kevpar suggested porting the code to hcsshim, since most it is VM-specific. Agree with making an overload, although technically nothing should depend on this code, since it's "new".

Reply:
ah, ok. hcsshim already depends on the signature haha. my oversight.

Reply:
Np